src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  health = 100;
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health, hidden = false) {
```

Comment:
The `health` parameter in the `Herbivore` constructor is unnecessary because the `Animal` class already initializes `health` to `100` by default. Remove this parameter to avoid confusion and potential bugs.